api/rosetta/data_block_integration_test.go

```diff
@@ -143,15 +143,15 @@ func TestAPI_Block(t *testing.T) {
 			err = api.Block(ctx)
 			assert.NoError(t, err)
 
-			// unpack response
+			// Unpack response.
```

Comment:
Just as a side note, I find these kind of comments superfluous. It's a bit like this:

```go
// unpack response
response, err := unpack(data)

// validate block
ok := validateBlock(response.Block)

// verify index
ok := validateIndex(response.Block.Index)
```

We should remove those imo. Comments should not describe the same thing that the code describes. They should be more about the how and why than the what.